src/Xamarin.Android.Tools.BootstrapTasks/Xamarin.Android.Tools.BootstrapTasks/UnzipDirectoryChildren.cs

```diff
@@ -29,6 +29,8 @@ public UnzipDirectoryChildren ()
 		[Required]
 		public  ITaskItem       DestinationFolder   { get; set; }
 
+		public bool NoSubdirectory { get; set; }
```

Comment:
I don't like this property name. Let's try to find a better one...

The reason the task is `UnzipDirectoryChildren` and not `Unzip` is precisely *because* it "skips"/"ignores" the topmost extracted directory, hence the `Directory.EnumerateDirectories(nestedTemp, "*")`.

In an ideal world, I think we'd want a separate task, but that's also ugly.

Perhaps `IncludeRootDirectory` would be clearer?

I'll I'm sure of is that `NoSubdirectory` doesn't seem clear.